README.md

```diff
@@ -6,6 +6,9 @@
 
 My goal here is to master all of the tools and best practices learned in previous microverse exercise like github flow and linters.
 
+## Online version
+
+-You can find the live demo on https://dagic-zewdu.github.io/portifolio/
 ## Built With
 
 - Lighthouse (An open-source, automated tool for improving the quality of web pages. It has audits for performance, accessibility, progressive web apps, SEO and more).
```

Comment:
[OPTIONAL]: It would be more professional if you can make some changes to your README,md file to show your live demo link like this as it is in the [README.md template](https://github.com/microverseinc/readme-template).
![demo](https://user-images.githubusercontent.com/10332499/149730838-5393d188-e424-4ee3-8ce8-64f2b1b380a0.PNG)